src/Vault/Vault.csproj

```diff
@@ -10,19 +10,19 @@
     <Company>Hashicorp</Company>
     <AssemblyTitle>Vault OpenAPI Library</AssemblyTitle>
     <Description>A Vault Client library generated from an OpenAPI doc</Description>
-    <Copyright>No Copyright</Copyright>
```

Comment:
we should update this over removing it
copyright headers got added in https://github.com/hashicorp/vault-client-dotnet/pull/96 (not sure what to put in this xml tho)

Reply:
Yeah. I wanted to remove it for now b/c it shows up when you publish it to Nuget. I don't want it saying "No copyright" there. But agreed. we should update it.

Reply:
I looked at a few other Nuget packages under Hashicorp and didn't see a copy right there, but I'll see what we should change it to